circle.yml

```diff
@@ -1,4 +1,6 @@
 machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
```

Comment:
(I don't have hopes for get the cache to work. Definitely not with 1.10 but 1.9 neither)